qutebrowser/browser/commands.py

```diff
@@ -709,6 +709,12 @@ def yank(self, what='url', sel=False, keep=False):
             sel = False
             target = "clipboard"
 
+        urlQuery = QUrlQuery(s)
+        for  key  in dict(urlQuery.queryItems()):
```

Comment:
Please only use one space before/after `key` here. Also, is the `dict()` needed? Isn't `urlQuery.queryItems()` a dict already?